NewsBank.js

```diff
@@ -0,0 +1,100 @@
+{
+	"translatorID": "7fc76bfc-3a1a-47e7-93cc-4deed69bee5f",
+	"label": "NewsBank",
+	"creator": "Reuben Peterkin",
+	"target": "https?://infoweb.newsbank.com/apps/news/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-08-06 07:05:28"
+}
+
+function detectWeb(doc, url) {
+	if (getRISText(doc)) return "newspaperArticle";
+	if (url.includes("results")) return "multiple";
+	//	Zotero.debug(url.indexOf("results"));
+	return false;
+}
+
+function getSearchResults(doc) {
+	var items = {}, found = false;
+	var rows = doc.getElementById('search-hits-gnus-search-hits-pane');
+	if (!rows) return false;
+	rows = rows.getElementsByTagName('article');
+	//	Zotero.debug(rows);
+
+	for (var i = 0; i < rows.length; i++) {
+		//		var count = rows[i].getElementsByClassName('count')[0];
+		//		if (!count) count = "";
+		//		else count = count.textContent.replace(/^\s*(\d+)[\s\S]*/, '$1') + '. ';
+
+		//		var title = doc.querySelector('.search-hits__hit__title');
+		var title = rows[i].getElementsByClassName('search-hits__hit__title')[0];
+		var hdl = rows[i].getElementsByTagName('a')[0];
+		var prefix = hdl.getElementsByClassName('element-invisible')[0];
+		if (!hdl) continue;
```

Comment:
```suggestion
		if (!title || !hdl) continue;
```

So that line 42 doesn't potentially cause an error, which it could right now.